### PR TITLE
Internal Sync for 1.6.0 Release (and CHANGELOG and version changes)

### PR DESCRIPTION
**Pull request by sid-github**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,33 @@
...
+* Has been tested for support till iOS 10.01 on devices and simulator.
+
+### Bug Fixes
+* Fixed CocoaPods issue with using EarlGrey as a module in Swift projects.
+* Fixed issue with Accessibility service not enabled for simulators and devices.
+* Minor documentation and syntax fixes.
+
+### Enhancements
+* Moved failure handler from a global variable to a thread local storage, like NSAssertionHandlers.
+* Exposed the angle for a GREYPinchAction.*
```
Comment (khandpur) [inline CHANGELOG.md]: remove trailing * here and the bullet below.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,33 @@
...
+* Added EarlGreyExample CocoaPods project to travis, deintegrating it for a clean setup.*
+
+### Deprecations
+* Deprecated `grey_pinchSlowInDirection` and `grey_pinchFastInDirection` in favor of
+  `grey_pinchSlowInDirectionWithAngle` and `grey_pinchFastInDirectionWithAngle`.
```
Comment (khandpur) [inline CHANGELOG.md]: we don't have any methods called `grey_pinchSlowInDirectionWithAngle` or `grey_pinchFastInDirectionWithAngle`

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,9 +47,12 @@ Baseline: [f3ee931]
 ```
 
 ### Compatibility
-* Handle bit-dependent CGFloat with bit-dependent trig function
+* Requires iOS 8 as the minimum deployment target.
+* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3.
+* Has been tested for support till iOS 10.01 on devices and simulator.
```
Comment (khandpur) [inline CHANGELOG.md]: we've tested for 10.1 as well.

File: CHANGELOG.md [hunk 4]
```diff
@@ -21,9 +47,12 @@ Baseline: [f3ee931]
...
 
 ### Enhancements
+* Handle bit-dependent CGFloat with bit-dependent trig function
```
Comment (khandpur) [inline CHANGELOG.md]: we didn't actually. we just cast double to CGFloat now.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,7 +3,33 @@
...
+
+```
+Baseline: [5080a21]
+   + [5080a21]: Updated changelog info.plist pod spec and gem version for 1.6.0 release.
+```
+
+### Compatibility
+* Requires iOS 8 as the minimum deployment target.
+* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3.
+* Has been tested for support till iOS 10.01 on devices and simulator.
```
Comment (khandpur) [inline CHANGELOG.md]: we've tested for 10.1 as well.

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,7 +3,33 @@
 Details changes in each release of EarlGrey. EarlGrey follows
 [semantic versioning](http://semver.org/).
 
-## [1.5.3](https://github.com/google/EarlGrey/tree/1.5.3) (11/14/2016)
+## [1.6.0](https://github.com/google/EarlGrey/tree/1.6.0)(12/06/2016)
```
Comment (khandpur) [inline CHANGELOG.md]: space before (12/06/2016)

File: EarlGrey/Common/GREYExposed.h [hunk 7]
```diff
@@ -359,3 +342,42 @@ IOHIDEventRef IOHIDEventCreateDigitizerFingerEvent(CFAllocatorRef allocator,
...
+                              notification:(CFStringRef)notification;
+
+@end
+
+/**
+ *  Used for enabling accessibility on device.
+ */
+@interface XCAXClient_iOS
+
+/*
```
Comment (khandpur) [inline EarlGrey/Common/GREYExposed.h]: /**

(sorry i missed it in my cl)

File: CHANGELOG.md (hunk 6 above)
Comment (sid-github) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (sid-github) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (sid-github) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (sid-github) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (sid-github) [inline CHANGELOG.md]: Done

File: EarlGrey/Common/GREYExposed.h (hunk 7 above)
Comment (sid-github) [inline EarlGrey/Common/GREYExposed.h]: Done

**Pull request merged**